index.html

```diff
@@ -18,14 +18,27 @@
                 </a>
             </div>
             <nav>
-                <a href="#" rel="noopener" target="_blank">
-                    <img src="/assets/images/breadcrumb.png" alt="">
-                </a>
+                <ul class="links-index">
+                    <li>
+                        <a href="#">Portfolio</a>
+                    </li>
+                    <li>
+                        <a href="#">About</a>
+                    </li>
+                    <li>
+                        <a href="#">Contact</a>
+                    </li>
+                </ul>
+                <div id="breadcrumb">
+                    <a href="#" rel="noopener" target="_blank">
+                        <img src="./assets/images/breadcrumb.png" alt="">
+                    </a>
+                </div>
             </nav>
         </div>
     </header>
     <section class="content">
-        <h1>I’m Alexandra. <br />Glad to see you!</h1>
+        <h1>I’m Javier. <br />Glad to see you!</h1>
         <div class="paragraph">I’m a software developer! I can help you build a product , feature or website Look
             through some of my work and experience! If you like what you see and have a project you need coded, dont
             hesitate to contact me.</div>
```

Comment:
- [ ] **[OPTIONAL]** Please consider fixing the background image for the desktop version. There is two background image on the figma design, one for mobile and one for desktop. Please make sure the desktop background is used on this section. This will make your project prettier and make the user know you're an awesome developer as you can precisely follow a professional web developer's design 😃 💪 

**Note:** This project requirement was marked as **[OPTIONAL]** by the previous reviewer so please consider implementing it.

Reply:
Ok, buddy Allasane @Trast00, then the background color on Desktop version is gray?? I'm a bit confused (I was thinking that we needed to use the first background as image for all the project :-/